crates/client-api/src/routes/database.rs

```diff
@@ -148,8 +146,6 @@ pub async fn call<S: ControlStateDelegate + NodeDelegate>(
             let (status, body) = reducer_outcome_response(&identity, &reducer, result.outcome);
             Ok((
                 status,
-                TypedHeader(SpacetimeIdentity(caller_identity)),
-                TypedHeader(SpacetimeIdentityToken(caller_identity_token)),
```

Comment:
to double-check: are these always added by a different layer of the stack right now? even for things that don't use the `auth_middleware`?

Reply:
No, only for `anon_auth_middleware` routes. However, the only routes that added the headers before now have `anon_auth_middleware`, so it's fine.